source/manual/business-readiness-publish-changes.html.md

```diff
@@ -0,0 +1,40 @@
+---
+owner_slack: "#govuk-platform-health"
+title: Publishing changes to the business readiness finder
+section: Business readiness finder
+layout: manual_layout
+parent: "/manual.html"
+important: true
+last_reviewed_on: 2019-01-18
+review_in: 3 months
+---
+
+The full journey for the  [business readiness finder][business-readiness-finder] is made up of the content items:
+
+1. The start page: [`/business-uk-leaving-eu`](https://www.gov.uk/business-uk-leaving-eu)
+2. The question pages: [`/prepare-business-uk-leaving-eu`](https://www.gov.uk/prepare-business-uk-leaving-eu)
+3. The finder: [`/find-eu-exit-guidance-business`](https://www.gov.uk/find-eu-exit-guidance-business)
+
+The content fo the start page can be updated in Mainstream Publisher as you would any other publisher.
+
+The content item for the questions pages doesn't actually contain any details. The titles of the questions are defined in a YAML file in [finder frontend](finder-frontend). However, the body of the question, for example the options if the question is a checkbox are read from the content item of the finder itself (content item 3 in the list). Finder frontend searches the content item for a facet "key" that matches the question, e.g. `sector_business_area` to find the question content.
+
+## Updating question titles
```

Comment:
For some reason this doesn't seem to be rendering as a heading

<img width="912" alt="screen shot 2019-01-18 at 15 53 20" src="https://user-images.githubusercontent.com/29889908/51397461-36eb8000-1b39-11e9-8fed-2d3ff80c79f4.png">